go/common/crypto/signature/signers/plugin/rpc.go

```diff
@@ -0,0 +1,103 @@
+package plugin
+
+import (
+	"net/rpc"
+
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
+)
+
+var _ Signer = (*rpcClient)(nil)
+
+type rpcServer struct {
+	impl Signer
+}
+
+// XXX: Do the arg types need to be public?
```

Comment:
Do they need to be? All the encoding/decoding is done in this package.

Reply:
Yes.